ethexe/service/src/tests/mod.rs

```diff
@@ -239,15 +237,15 @@ async fn uninitialized_program() {
     // Case #1: Init failed due to panic in init (decoding).
     {
         let res = env
-            .create_program(code_id, 500_000_000_000_000)
+            .create_program(code_id, H256::zero(), None, 500_000_000_000_000)
```

Comment:
I'd split the most used call and call with full arguments and reused second in first one